common/web/types/src/ldml-keyboard/pattern-parser.ts

```diff
@@ -340,13 +337,24 @@ export class MarkerParser {
      * subfunc: add all markers in the pending (last_markers) queue
      * @param l string the marker is 'glued' to, or '' for end
      */
-    function add_pending_markers(l: string): void {
-      // first char, or, marker-before-eot. Must be first char of NFD sequence
-      const ch = (l === '') ? MARKER_BEFORE_EOT : [...(l.normalize("NFD"))][0];
+    function add_pending_markers(l: string){
+      // first char, or, marker-before-eot
+      const glueChars = (l === '') ? [MARKER_BEFORE_EOT] : [...(l.normalize("NFD"))];
+      const glue = glueChars[0];
+      // push the 'end' value
+      map.push({ ch: glue, end: true });
       while(last_markers.length) {
         const marker = last_markers[0];
         last_markers = last_markers.slice(1); // pop from front
-        map.push({ ch, marker });
+        map.push({ ch: glue, marker });
+      }
+      // now, push the rest of the glue chars as an NFD sequence.
+      // For example, `\m{m}\u0344` will create the following stream:
+      //  { ch: 0308, end: true}
+      //  { ch: 0308, marker: 1}
+      //  { ch: 0301, end: true}  // added because of decomp
```

Comment:
I see so the marker gets glued to the last character in the decomposition. Definitely slightly unintuitive -- more so than other gluings.

Reply:
Should be FIRST char

Reply:
I'm not entirely following?

`\m{m}\u0344` ==> `\m{m}\u0308\u0301` or `\u0308\m{m}\u0301`?

Reply:
`\m{m}\u0308\u0301` - adding a test.  markers are re-added in reverse order.